js/adapt-languagePicker.js

```diff
@@ -42,20 +42,12 @@ define([
   function onOfflineStorageReady() {
     var previousLanguage = Adapt.offlineStorage.get("lang");
```

Comment:
sorry I know I'm being sadly picky here but could this be changed to `storedLanguage`?